crates/tuono/src/scaffold_project.rs

```diff
@@ -151,6 +161,11 @@ pub fn create_new_project(
 
     update_package_json_version(&folder_path).expect("Failed to update package.json version");
     update_cargo_toml_version(&folder_path).expect("Failed to update Cargo.toml version");
+
+    if git {
+        init_new_git_repo(&folder_path).expect("Failed to initialise a new git repo");
```

Comment:
We should never make the command panic

```suggestion
        init_new_git_repo(&folder_path).unwrap_or_else(|| exit_with_error("Failed to initialise a new git repo"));
```